<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registers</class>
 <widget class="QWidget" name="registers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>91</width>
     <height>416</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Monospace</family>
       </font>
      </property>
      <property name="text">
       <string>A</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="reg_A">
      <property name="font">
       <font>
        <family>Monospace</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0x0000</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Monospace</family>
       </font>
      </property>
      <property name="text">
       <string>B</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="reg_B">
      <property name="font">
       <font>
        <family>Monospace</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0x0000</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Monospace</family>
       </font>
      </property>
      <property name="text">
       <string>C</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="reg_C">
      <property name="font">
       <font>
        <family>Monospace</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0x0000</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>Monospace</family>
       </font>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <family>Monospace</family>
       </font>
      </property>
      <property name="text">
       <string>Y</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Monospace</family>
       </font>
      </property>
      <property name="text">
       <string>Z</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <family>Monospace</family>
       </font>
      </property>
      <property name="text">
       <string>I</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <family>Monospace</family>
       </font>
      </property>
      <property name="text">
       <string>J</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="reg_X">
      <property name="font">
       <font>
        <family>Monospace</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0x0000</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="reg_Y">
      <property name="font">
       <font>
        <family>Monospace</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0x0000</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="reg_Z">
      <property name="font">
       <font>
        <family>Monospace</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0x0000</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="reg_I">
      <property name="font">
       <font>
        <family>Monospace</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0x0000</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="reg_J">
      <property name="font">
       <font>
        <family>Monospace</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0x0000</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_9">
      <property name="font">
       <font>
        <family>Monospace</family>
       </font>
      </property>
      <property name="text">
       <string>PC</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_10">
      <property name="font">
       <font>
        <family>Monospace</family>
       </font>
      </property>
      <property name="text">
       <string>SP</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_11">
      <property name="font">
       <font>
        <family>Monospace</family>
       </font>
      </property>
      <property name="text">
       <string>EX</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_12">
      <property name="font">
       <font>
        <family>Monospace</family>
       </font>
      </property>
      <property name="text">
       <string>IA</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLineEdit" name="reg_SP">
      <property name="font">
       <font>
        <family>Monospace</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0x0000</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="reg_PC">
      <property name="font">
       <font>
        <family>Monospace</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0x0000</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QLineEdit" name="reg_EX">
      <property name="font">
       <font>
        <family>Monospace</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0x0000</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QLineEdit" name="reg_IA">
      <property name="font">
       <font>
        <family>Monospace</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0x0000</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_run">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>94</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_step">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>94</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Step</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_stop">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>94</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_pause">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>94</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
